src/App.tsx

```diff
@@ -24,69 +97,54 @@ export const App: React.FC = () => {
           className="toggle-all"
           data-cy="toggleAll"
         />
-        <label htmlFor="toggle-all">Mark all as complete</label>
 
-        <ul className="todo-list" data-cy="todoList">
-          <li>
-            <div className="view">
-              <input type="checkbox" className="toggle" id="toggle-view" />
-              <label htmlFor="toggle-view">asdfghj</label>
-              <button type="button" className="destroy" data-cy="deleteTodo" />
-            </div>
-            <input type="text" className="edit" />
-          </li>
-
-          <li className="completed">
-            <div className="view">
-              <input type="checkbox" className="toggle" id="toggle-completed" />
-              <label htmlFor="toggle-completed">qwertyuio</label>
-              <button type="button" className="destroy" data-cy="deleteTodo" />
-            </div>
-            <input type="text" className="edit" />
-          </li>
-
-          <li className="editing">
-            <div className="view">
-              <input type="checkbox" className="toggle" id="toggle-editing" />
-              <label htmlFor="toggle-editing">zxcvbnm</label>
-              <button type="button" className="destroy" data-cy="deleteTodo" />
-            </div>
-            <input type="text" className="edit" />
-          </li>
-
-          <li>
-            <div className="view">
-              <input type="checkbox" className="toggle" id="toggle-view2" />
-              <label htmlFor="toggle-view2">1234567890</label>
-              <button type="button" className="destroy" data-cy="deleteTodo" />
-            </div>
-            <input type="text" className="edit" />
-          </li>
-        </ul>
+        <TodoList todos={filteredTodos} />
       </section>
 
       <footer className="footer">
         <span className="todo-count" data-cy="todosCounter">
-          3 items left
+          {`${incompletedTodosCount} items left`}
         </span>
 
         <ul className="filters">
           <li>
-            <a href="#/" className="selected">All</a>
+            <a
+              href="#/"
+              className="selected"
+              onClick={() => handleFilterChange(Status.All)}
+            >
+              All
+            </a>
```

Comment:
add className="selected" to your items depends on current status.
also can you check some similar things in this link items, create some array, and iterate through it?
like
```
const VALUES = [
  { path: '#/', filterValue: Status.All, label: 'All' },
  { path: '#/active', filterValue: Status.Active, label: 'Active' },
  { path: '#/completed', filterValue: Status.Completed, label: 'Completed' },
]


jsx
{VALUES.map(item => (
  <li key={item.path}>
    <a
      href={item.path}
      className={classNames({
        selected: filterField === item.filterValue,
      })}
      onClick={(event) => handleClick(event, item.filterValue)}
    >
      {item.label}
    </a>
  </li>
))}
```

follow the DRY principle (Don't Repeat Yourself)